### DISCUSSION TITLE
关于启动器的启动参数

### DISCUSSION THREAD

**Discussion by sileence114** [Meloong-Git/PCL#3450] [提问] [answered]
我想实现：
pcl.exe --run_version "1.20.4"
通过命令行直接启动某个版本，而无需打开启动器再次点启动游戏。
我想将这个保存为快捷方式直接双击运行，或者启动器生成一个快捷方式？
（导出启动脚本的话，会不会因为token过期导致启动失败？）

**Comment by zkitefly** [comment, marked as answer]
已被拒

https://github.com/Hex-Dragon/PCL2/issues/2343
